docs/user-guide/commands.md

```diff
@@ -296,43 +296,68 @@ Example:
 
     $ watson report
     Mon 05 May 2014 -> Mon 12 May 2014
-    
+
     apollo11 - 13h 22m 20s
             [brakes    7h 53m 18s]
             [module    7h 41m 41s]
             [reactor   8h 35m 50s]
             [steering 10h 33m 37s]
             [wheels   10h 11m 35s]
-    
+
     hubble - 8h 54m 46s
             [camera        8h 38m 17s]
             [lens          5h 56m 22s]
             [transmission  6h 27m 07s]
-    
+
     voyager1 - 11h 45m 13s
             [antenna     5h 53m 57s]
             [generators  9h 04m 58s]
             [probe      10h 14m 29s]
             [sensors    10h 30m 26s]
-    
+
     voyager2 - 16h 16m 09s
             [antenna     7h 05m 50s]
             [generators 12h 20m 29s]
             [probe      12h 20m 29s]
             [sensors    11h 23m 17s]
-    
+
     Total: 43h 42m 20s
-    
+
     $ watson report --from 2014-04-01 --to 2014-04-30 --project apollo11
     Tue 01 April 2014 -> Wed 30 April 2014
-    
+
     apollo11 - 13h 22m 20s
             [brakes    7h 53m 18s]
             [module    7h 41m 41s]
             [reactor   8h 35m 50s]
             [steering 10h 33m 37s]
             [wheels   10h 11m 35s]
 
+    $ watson report --format json
```

Comment:
Here you document `--format json` while the actual option is `--json`.
BTW, I think the `--format` is better since it leaves room to future other formats without adding lots of distinct options.